functions/main.py

```diff
@@ -68,7 +68,7 @@ def get_contents(document_data: Dict[str, Any]) -> Dict[str, str]:
         if blob is None:
             continue
         size += blob.size or 0
-        if (size / 1e6) > MAX_SIZE:
+        if (size / 1e6) > max_size_megabytes:
             break
```

Comment:
I think there should be an error here being raised, in order for the `create_tree_v2` function to capture it, and mark the process as failed, adding a user friendly there as well, so that the user will receive a detailed feedback on what they should do, in order for them to be able to do another attempt.

It seems like this will return an empty "content" making the tree extraction to fail, but will still be a "hidden error".